lxd/storage_pools_config.go

```diff
@@ -128,30 +132,14 @@ func storagePoolFillDefault(name string, driver string, config map[string]string
 		config["size"] = strconv.FormatUint(uint64(size), 10)
 	}
 
-	if driver == "lvm" {
-		if config["lvm.thinpool_name"] == "" {
-			config["lvm.thinpool_name"] = "LXDThinpool"
-		}
-
-		if config["volume.block.filesystem"] == "" {
-			config["volume.block.filesystem"] = "ext4"
-		}
-
-		if config["volume.block.mount_options"] == "" {
-			config["volume.block.mount_options"] = "discard"
-		}
-	}
-
-	if config["volume.size"] == "" {
-		if driver == "lvm" {
-			sz, err := shared.ParseByteSizeString("10GB")
+	if driver == "lvm" || driver == "btrfs" || driver == "zfs" {
+		if config["volume.size"] != "" {
```

Comment:
Yeah, so we should probably only use this when `lvm` is set and only update the quota property via the root disk device?